docs/reference/channels.rst

```diff
@@ -0,0 +1,61 @@
+.. _channels:
+
+Channels
+========
+
+Snaps are published on *channels*, which are concurrent versions of the snap,
+addressing needs such as preview releases and long-term version support.
+
+A channel is composed of three components -- a track, a risk, and a branch.
+
+Briefly, a channel is represented in Snapcraft like this:
+
+  <track>/<risk>/<branch>
+
+In Snapcraft commands, a typical channel without a branch looks like this:
+
+.. code:: bash
+
+  --channel=latest/edge
+
+
+Track
```

Comment:
Is it worth linking to https://snapcraft.io/docs/process-for-aliases-auto-connections-and-tracks? It's not obvious that tracks must be requested.

Reply:
1df1efa25

I'm concerned this new doc isn't sufficient, though. We haven't explained the track policy, especially in light of our statement that they're up to the author's discretion. What's the skinny on custom tracks?

Reply:
I should have been clearer: I'm concerned the copy in this PR is misleading. The statement

>  the release convention for tracks is at the author's discretion

could lead readers to think that they can publish custom tracks as they please.

What's the breakdown of tracks? Latest and stable, and everything else is by request?

Reply:
> Latest is the only default preset, and all others are by request

Correct.

> the author's discretion

And by this, I think it means an author could make a forum request for a track with the name of their choice.

Reply:
07bd6c985